src/perftest_parameters.c

```diff
@@ -961,9 +961,15 @@ static void force_dependecies(struct perftest_parameters *user_param)
 	}
 
 	if (user_param->post_list > 1) {
-		user_param->cq_mod = user_param->post_list;
-		printf(RESULT_LINE);
-		printf("Post List requested - CQ moderation will be the size of the post list\n");
+		if (user_param->cq_mod == DEF_CQ_MOD) {
```

Comment:
May be it worths checking that CQ_mod was explicitly set to DEF_CQ_MOD, but not chosen by default with a flag or smth.